lang/lua/astal/gtk3/app.lua

```diff
@@ -24,30 +24,28 @@ end
 
 local app = AstalLua()
 
----@class StartConfig
----@field icons? string
----@field instance_name? string
----@field gtk_theme? string
----@field icon_theme? string
----@field cursor_theme? string
----@field css? string
----@field hold? boolean
----@field request_handler? fun(msg: string, response: fun(res: any))
----@field main? fun(...): unknown
----@field client? fun(message: fun(msg: string): string, ...): unknown
+--- @alias StartConfig {
```

Comment:
should not this be `---@alias` without a space inbetween?
also, I'm getting a lot "Fields cannot be injected in to the reference of StartConfig" with an alias

Reply:
yeah, it should be without the space, also for some reason, Zed works with that type of annotations, while lua-language-server doesnt

![image](https://github.com/user-attachments/assets/2e120db8-f1d6-45fe-85e7-bc2d255d9cdb)